AI/VCAI/VCAI.h

```diff
@@ -138,7 +141,10 @@ class VCAI : public CAdventureAI
 
 	friend class FuzzyHelper;
 
+	std::map<TeleportChannelID, shared_ptr<TeleportChannel> > knownTeleportChannels;
 	std::map<const CGObjectInstance *, const CGObjectInstance *> knownSubterraneanGates;
+	ObjectInstanceID destinationTeleport;

```

Comment:
This one is not serialized, here should be a comment about that or you missed serialization?